zoo-css-challenge/style.css

```diff
@@ -187,9 +235,31 @@ footer {
 .membership strong {
   color: #ff7f11;
 }
-
+.membership p {
+  max-width: 700px;
+  margin-right: 200px;
+  margin-left: 250px;
+}
 /*Programs*/
 
-.programs .card {
+.programs .bottles {
   display: block;
 }
+.bottles {
```

Comment:
There's no `bottles` class in the HTML document, so this doesn't do anything and therefore we should delete it